packages/foreman/foreman/foreman.spec

```diff
@@ -830,6 +809,15 @@ Meta package with support for plugins.
 %{_sysconfdir}/rpm/macros.%{name}-plugin
 %{_datadir}/%{name}/schema_plugin.rb
 
+%package plugin-build
```

Comment:
We already have `foreman-plugin` package that has all the build support for a Foreman plugin.

Reply:
We do, but given this is a special macro that needs to live in the buildroot, I'm worried about a chicken'n'egg situation that could be simplified by just having this simple package with the macro.